Android/app/src/androidTest/java/PageObjects/LoginPageObject.kt

```diff
@@ -0,0 +1,80 @@
+package PageObjects
+
+import android.os.Build
+import android.support.test.uiautomator.UiSelector
+import android.util.Log
+
+/**
+ * Created by bpage on 2/21/18.
+ */
+class LoginPageObject : BasePageObject() {
+
+    init {
+        if (isOldDevice) {
+            device.findObject(UiSelector().className("android.widget.EditText").index(2)).waitForExists(120000)
+        }
+    }
+
+    fun setUsername(name: String) {
+        val usernameField = if (isOldDevice) {
+            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
Could you add `FIXME` wherever these conditionals exist to remove them once our `minVersion` is greater than the version specified?